conan/tools/cmake/toolchain.py

```diff
@@ -425,34 +439,64 @@ def context(self):
         if prefer_config is not None and prefer_config.lower() in ("false", "0", "off"):
             find_package_prefer_config = "OFF"
 
-        os_ = self._conanfile.settings.get_safe("os")
-        android_prefix = "${CMAKE_CURRENT_LIST_DIR}" if os_ == "Android" else None
```

Comment:
This was necessary at some point, to have the Android build finding the xxx-config.cmake by Conan. Maybe it is no longer needed, but it is difficult to know, because we are not running yet Android build tests in our CI

Reply:
Yes, it's the same issue than cross-build to iOS/tvOS/watchOS, but 2 different solutions have been implemented. Here it's fixed by forcing `CMAKE_FIND_ROOT_PATH_MODE_PACKAGE` to `"BOTH"` (same fix than iOS/tvOS/watchOS).